vite.config.js

```diff
@@ -23,12 +23,28 @@ const manifest = {
     ]
 };
 
+function externalCSSPlugin() {
+    return {
+        name: 'external-userstyles',
+        transformIndexHtml: {
+            enforce: 'post',
+            transform() {
+                return [{
+                    tag: "link",
+                    attrs: {"rel": "stylesheet", "type": "text/css", "href": "/userstyles.css"},
```

Comment:
Injects the empty `userstyles.css` to the end of the HTML head element, allowing users to inject their own styles (without having to `!important;` everything)